### DISCUSSION TITLE
请教如何部署到 Cloudflare Tunnel

### DISCUSSION THREAD

**Discussion by xihaoxiaohao** [1715173329/UnblockNeteaseMusic#1281] [Q&A]
### 需求描述

请问有没有办法把UnblockNeteaseMusic server安装到家里的树莓派，然后安装Cloudflare Tunnel 把域名指向家里局域网的UnblockNeteaseMusic server，从而实现在公网使用UnblockNeteaseMusic server？

### 预期行为

部署到 Cloudflare Tunnel从而实现在公网使用UnblockNeteaseMusic server。

### 网易云音乐版本号

Version 2.3.14 (999)

### 操作系统

MAC OS

**Comment by wukibaka** [comment]
可以试试 cloudflare相当于是一个frp 可以去看看[官方文档](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/)自行尝试
